common-npm-packages/packaging-common/locationUtilities.ts

```diff
@@ -167,6 +167,27 @@ export async function retryOnExceptionHelper<T>(action: () => Promise<T>, maxTri
     }
 }
 
+export async function retryOnNullOrExceptionHelper<T>(action: () => Promise<T>, maxTries: number, retryIntervalInMilliseconds: number): Promise<T> {
+    while (true) {
+        try {
+            var response = await action();
+            if(!response) {
```

Comment:
I suppose nothing restricts action to return `false` as a valid response?  Maybe `response == null` is a better condition? It both covers `null` and `undefined`, but allows `false`.

Reply:
I initially had this hideous thing: `response === null || response === undefined`.  I'll switch it back!